packages/ui/src/Button/ButtonIcon.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+export interface ButtonIconProps {
+    icon: React.ReactNode;
+}
+
+export const ButtonIcon = (props: ButtonIconProps) => <>{props.icon}</>;
```

Comment:
As we discussed, we need the `Icon` component. I would continue to use the `<Icon />` component from `@webiny/ui`. 

Internally we will re-wire it to the new component.

Reply:
Fixed.